cmd/ocitools/validate.go

```diff
@@ -84,7 +84,7 @@ var bundleValidateCommand = cli.Command{
 			return fmt.Errorf("The root path %q is not a directory.", rootfsPath)
 		}
 
-		hostCheck := context.Bool("host-specific")
+		hostCheck := hostSpecific || context.Bool("host-specific")
```

Comment:
> I don't think we want both a global and a per-command --host-specific here. We should drop the per-command option.

 I am okay with drop the per-command --host-specific option once @mrunalp  and you are okay with it. 
I keep it for now just in case some important users are using it.